README.md

```diff
@@ -1,26 +1,17 @@
-Amiable
-=======
+# Amiable
 Amiable is a web app for monitoring the use of AMIs.
 You can access it here:
 https://amiable.gutools.co.uk
 
 
-### How to run Amiable locally
+## How to run Amiable locally
+Amiable uses Google Auth. For this reason, we need to run amiable through an nginx proxy `amiable.local.dev-gutools.co.uk`. 
```

Comment:
Minor: `amiable` should start with a capital letter here

Reply:
Ha @katebee, just wanted to point out that there's quite a nice GitHub ['Suggestion' feature](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) for suggesting changes like these!

![image](https://user-images.githubusercontent.com/52038/81667733-8d21b800-943b-11ea-8656-a3440570cec3.png)

Reply:
thank you, I was not aware! 🥇